cni/pkg/nodeagent/net.go

```diff
@@ -287,7 +293,8 @@ func (s *NetServer) syncHostIPSets(ambientPods []*corev1.Pod) error {
 		} else {
 			err := addPodToHostNSIpset(pod, podIPs, &s.hostsideProbeIPSet)
 			if err != nil {
-				return err
+				// nolint: lll
+				log.Errorf("pod %s has IPs %v, but previously inserted one of those IPs for a different pod, this indicates an IPAM problem, pod will be skipped and will fail healthchecks", pod.Name, podIPs)
```

Comment:
nit: we don't really know the error is due to duplicate IPs, do we? Also we already log an error in addPodToHostNSIpset so now we would double log, maybe we can just have it return an error and we log that

Reply:
Yeah the actual ipset error further down would indicate that. I erred on the side of making more noise, so it's easier to identify what happened, if it does.

Reply:
Fixed this a bit